doc/source/whatsnew/v0.23.0.txt

```diff
@@ -555,6 +555,7 @@ Sparse
 
 - Bug in which creating a ``SparseDataFrame`` from a dense ``Series`` or an unsupported type raised an uncontrolled exception (:issue:`19374`)
 - Bug in :class:`SparseDataFrame.to_csv` causing exception (:issue:`19384`)
+- Bug in constructing a :class:`SparseArray`: if `data` is a scalar and `index` is defined it will coerce to float64 regardless of scalar's dtype. (:issue:`19163`)
```

Comment:
Whoops, just noticed we don't have `SparseArray` in our API docs so this link won't work. Just

    ``SparseArray``

until we get that sorted out. Also double ticks around data nd index.